### PR TITLE
Improve books page

### PR DESCRIPTION
**Pull request by bt-s**

### CODE REVIEW

File: src/components/AddNewBookForm/AddNewBookForm.js [hunk 1]
```diff
@@ -48,18 +45,18 @@ const AddNewBookFormBase = props => {
...
-        owner: JSON.parse(localStorage.getItem('authUser')).uid,
+        owner: props.authUser.username,
+        avatar: props.authUser.photoURL,
         rating: parseInt(rating),
         description,
         author,
         location,
         imageUrls,
-        type: 'lend',
+        type,
```
Comment (Envl) [inline src/components/AddNewBookForm/AddNewBookForm.js:55]: Here must be a type, either lend or give

File: src/components/Authentication/SignIn.js [hunk 2]
```diff
@@ -62,7 +61,6 @@ const SignInFacebookBase = props => {
     <form onSubmit={onSubmit} className="sign-in-facebook-form">
       <Button
         className="btn btn-auth btn-facebook"
-        type="submit"
         icon={<FontAwesomeIcon icon={faFacebookF} />}
```
Comment (Envl) [inline src/components/Authentication/SignIn.js]: This is missing.

**Review by Envl — APPROVED** [review]
I've commented out that missing faFacebookicon. You have to add it back before merging.

**Comment by Envl** [comment]
It looks much better! Great work Bas!

File: src/components/AddNewBookForm/AddNewBookForm.js (hunk 1 above)
Comment (bt-s) [reply src/components/AddNewBookForm/AddNewBookForm.js:55]: I think this works fine like it is, since I have made some changes. I can show you tomorrow!

**Pull request merged**